spec/query_spec.cr

```diff
@@ -737,4 +737,28 @@ describe Avram::Query do
       companies.first.should eq company
     end
   end
+
+  describe "#group" do
+    it "groups" do
+      user = UserBox.create &.age(25).name("Michael")
+      user = UserBox.create &.age(25).name("Dwight")
+      user = UserBox.create &.age(21).name("Jim")
+
+      users = UserQuery.new.group(&.age).group(&.id)
+      users.query.statement.should eq "SELECT #{User::COLUMN_SQL} FROM users GROUP BY users.age, users.id"
+      users.map(&.name).should contain "Dwight"
```

Comment:
I think there should also be an “eq” test to make sure the result is exactly what is expected